src/protocols/secure_channel/CASESession.cpp

```diff
@@ -646,6 +670,79 @@ CHIP_ERROR CASESession::RecoverInitiatorIpk()
     return CHIP_NO_ERROR;
 }
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+void CASESession::HandleConnectionComplete(Transport::ActiveConnectionState * conn, CHIP_ERROR err)
```

Comment:
What happens if connection setup fails?  Is this the method that gets called?  If so, it's not named very well....  Should be HandleConnectionAttemptComplete or something.

Reply:
Yes, this method gets called on setup failure. Renamed it.